### PR TITLE
feat(conference): add spotlight and unspotlight to Roster

### PR DESCRIPTION
**Pull request by drymarau**

### CODE REVIEW

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Roster.kt [hunk 1]
```diff
@@ -91,6 +91,22 @@ public interface Roster {
...
+     */
+    public suspend fun spotlight(participantId: UUID? = null): Unit = throw NotImplementedError()
+
+    /**
+     * Disables the "spotlight" on a participant or self.
+     *
+     * @param participantId an ID of the participant, null for self
+     * @throws UnspotlightException if the operation failed
+     */
+    public suspend fun unspotlight(participantId: UUID? = null): Unit = throw NotImplementedError()
```
Comment (tpettersen) [inline sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Roster.kt:108]: is unspotlight a valid word? Only reference I can find is in Zoom libs, other than that it is not in any dictionary afaik. Perhaps rename to `removeSpotlight`, `disableSpotlight` or similar?

File: sdk-conference/src/test/kotlin/com/pexip/sdk/conference/infinity/internal/RosterImplTest.kt [hunk 2]
```diff
@@ -881,6 +883,212 @@ class RosterImplTest {
...
+            event = event,
+            participantId = participantId,
+            store = store,
+            step = object : InfinityService.ConferenceStep {},
+        )
+        roster.unspotlight(UUID.randomUUID())
+    }
+
+    @Test
+    fun `unspotlight() throws SpotlightException`() = runTest {
```
Comment (tpettersen) [inline sdk-conference/src/test/kotlin/com/pexip/sdk/conference/infinity/internal/RosterImplTest.kt:1002]:
```suggestion
    fun `unspotlight() throws UnspotlightException`() = runTest {
```

**Review by tpettersen — APPROVED** [review]
Looks ok 👍 Just unsure of the naming

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Roster.kt (hunk 1 above)
Comment (drymarau) [reply sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Roster.kt:108]: Prefer to keep it short, as you mentioned there's at least one other library that uses it for a similar purpose so it should be OK.

**Pull request merged**